packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -122,6 +122,7 @@
   "components.Input.error.attribute.sameKeyAndName": "Can't be equal",
   "components.Input.error.validation.minSupMax": "Can't be superior",
   "components.Input.error.custom-error": "{errorMessage} ",
+  "components.Input.error.validation.json": "This is not a JSON",
```

Comment:
`This doesn't match with the JSON format`?

Reply:
+1 to that @Aurelsicoko though I would use `This doesn't match the JSON format`